cw/simple_capture_aes_sca.py

```diff
@@ -238,16 +265,37 @@ def abort_handler_during_loop(this_project, sig, frame):
 
     # Create and show test plot ------------------------------------------------
     # Use this plot to check for clipping and adjust gain appropriately
+    if CWLib:
+        proj_waves = project.waves
+    else:
+        proj_waves = project.get_waves_int()
+
     if cfg["capture"]["show_plot"]:
-        plot.save_plot_to_file(project.waves, None, cfg["capture"]["plot_traces"],
+        plot.save_plot_to_file(proj_waves, None, cfg["capture"]["plot_traces"],
                                cfg["capture"]["trace_image_filename"], add_mean_stddev=True)
         print(f'Created plot with {cfg["capture"]["plot_traces"]} traces: '
               f'{Path(cfg["capture"]["trace_image_filename"]).resolve()}')
 
-    # Save metadata and entire configuration cfg to project file ---------------
-    project.settingsDict['datetime'] = datetime.now().strftime("%m/%d/%Y, %H:%M:%S")
-    project.settingsDict['cfg'] = cfg
     if HUSKY:
         sample_rate = int(round(cwfpgahusky.scope.clock.adc_freq, -6))
+        sample_offset = cwfpgahusky.scope.adc.offset
```

Comment:
IIRC `adc.offset` is just the positive horizontal offset. If you want to record earlier (i.e. negative horizontal offset) this is stored under `adc.pre_samples` or somenthing. Alternatively, you can use the `offset_samples` variable from the OpenTitan device class.